### PR TITLE
Release Proposal for Version 1.0.0

### PR DESCRIPTION
**Pull request by DominicKramer**

### CODE REVIEW

**Comment by DominicKramer** [comment]
PTAL

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,23 @@
 # Node.js Agent for Google Cloud Debug ChangeLog
 
+## 2017-03-14, Version 1.0.0, @dominicdkramer
```
Comment (DominicKramer) [inline CHANGELOG.md]: Should I mark this as experimental?

Comment (matthewloring) [inline CHANGELOG.md]: Yup, we will go non-experimental with a beta release

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+
+This release marks the first beta release of the Stackdriver Debugger agent for Node.js.
```
Comment (matthewloring) [inline CHANGELOG.md]: Not beta yet.

File: CHANGELOG.md (hunk 1 above)
Comment (DominicKramer) [inline CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (DominicKramer) [inline CHANGELOG.md]: I removed the ß.

Comment (DominicKramer) [inline CHANGELOG.md]: I've also added a semver-major changes section.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,28 @@
 # Node.js Agent for Google Cloud Debug ChangeLog
 
+## 2017-03-14, Version 1.0.0 (Experimental), @dominicdkramer
+
+### Semver-major changes
+
+* Evaluation of expressions is no longer allowed by default.  It can be enabled using the `allowExpressions` configuration option.
+* When the agent is used with a Google Cloud Functions function, the function name is used as part of the debugger's description.
+* Configuration options can now be specified directly in the `options` object passed to the agent's `start` method.  Previously, the configuration options needed to be specified within `options.debug` in the `options` object passed to the `start` method.
+* An issue has been fixed where it was possible that some configuration options would not correctly use their default values.
```
Comment (ofrobots) [inline CHANGELOG.md]: This is not semver major.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,28 @@
 # Node.js Agent for Google Cloud Debug ChangeLog
 
+## 2017-03-14, Version 1.0.0 (Experimental), @dominicdkramer
+
+### Semver-major changes
+
+* Evaluation of expressions is no longer allowed by default.  It can be enabled using the `allowExpressions` configuration option.
+* When the agent is used with a Google Cloud Functions function, the function name is used as part of the debugger's description.
+* Configuration options can now be specified directly in the `options` object passed to the agent's `start` method.  Previously, the configuration options needed to be specified within `options.debug` in the `options` object passed to the `start` method.
```
Comment (ofrobots) [inline CHANGELOG.md]: not semver major

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,28 @@
 # Node.js Agent for Google Cloud Debug ChangeLog
 
+## 2017-03-14, Version 1.0.0 (Experimental), @dominicdkramer
+
+### Semver-major changes
+
+* Evaluation of expressions is no longer allowed by default.  It can be enabled using the `allowExpressions` configuration option.
+* When the agent is used with a Google Cloud Functions function, the function name is used as part of the debugger's description.
```
Comment (ofrobots) [inline CHANGELOG.md]: not semver major

Comment (DominicKramer) [inline CHANGELOG.md]: I thought all of these could possibly break previous code, and so I added them just to be safe and to get a second look.  I have now removed them except for the change to the evaluation of expressions.

**Pull request merged**